tutorials/celestia-node-key.md

```diff
@@ -162,6 +162,122 @@ then enter your bip39 mnemonic:
 ./cel-key --help
 ```
 
+## Managing the key used by celestia state account-address
+
+When you run a Celestia node and use the command `celestia state account-address`, it returns the address of the key that your node is currently using. This key is the same one that was either:
+
+1. Auto-generated during node initialization with `celestia <node-type> init`
+2. Manually created using `cel-key add` 
+3. Imported/recovered using `cel-key add --recover`
+
+### Understanding the relationship
+
+The key reported by `celestia state account-address` is stored in your node's keyring directory:
+- For Mocha testnet: `~/.celestia-<node-type>-mocha-4/keys/keyring-test/`
+- For Mainnet Beta: `~/.celestia-<node-type>/keys/keyring-test/`
+
+You can verify this by listing your keys with `cel-key`:
+
+```bash
+cel-key list --keyring-backend test --node.type <node-type> --p2p.network <network>
+```
+
+For example, with a light node on Mocha:
+
+```bash
+cel-key list --keyring-backend test --node.type light --p2p.network mocha
+```
+
+The address shown in this output should match the one returned by `celestia state account-address`.
+
+### Backing up your key
+
+To back up the key used by your node, you have two options:
+
+#### Option 1: Export the key's mnemonic
```

Comment:
Does this export the mnemonic or a key file? I think we can get this merged after clearing this up @hexmarkd